_includes/tutorials/ksql-heterogeneous-json/ksql/markup/answer/varchar-stream.adoc

```diff
@@ -0,0 +1,5 @@
+Create a stream and define the outer-most element of the JSON structures as `VARCHAR`
```

Comment:
To me I would think the answer is to use `EXTRACTJSONFIELD` which requires the JSON you're accessing to be a "string/varchar".

Reply:
Unless I misunderstand your point here, `EXTRACTJSONFIELD` always returns a string, but inside the JSON, the value you're accessing could be a non-string value.  I have an example of this in the tutorial.

Reply:
What I'm trying to get at is 
>If the requested JSONPath does not exist, the function returns NULL. (https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/scalar-functions/#extractjsonfield)

Reply:
@bbejeck  looking back at this. I still standby my initial responses but I think my initial point was lost. I think the short answer should have some mention of `EXTRACTJSONFIELD` rather than just having the JSON be a `VARCHAR`. `EXTRACTJSONFIELD` implies that the value you are extracting from is a `VARCHAR`, whereas just varchar doesn't imply anything about how one would actually retrieve data from the JSON. Feel free to disagree :) This is my only hangup, happy to give this my approval once this is resolved.

Reply:
@awalther28 I misunderstood what you were saying before, but I think I get it now.  Check out the short answer now and let me know